biogrid/libs/biogrid-simulator/src/lib/biobattery/biobattery-class.ts

```diff
@@ -24,7 +24,7 @@ export class BioBattery implements Battery {
 
   supplyPower(outputenergy: Energy): Energy {
     if (this.currentBatteryPower - outputenergy < 0) {
-      throw new Error(`Battery has less than ${outputenergy} units`);
+      throw `Battery has less than ${outputenergy} units`;
```

Comment:
Good use of including output energy here, maybe we can include this.currentBatteryPower as well.
Something like: "Requested to supply ___ power. Battery only has ___ units."